### PR TITLE
Url extraction v2

### PR DESCRIPTION
**Pull request by melamedbn**
Ready

## Description
fixes typos and grammar.

### CODE REVIEW

File: docs/indicators/domain-extraction.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+id: domain-extraction
+title: Domain Extraction
+---
+
+The Cortex XSOAR Domain indicator type is built using regex and a formatting script.
```
Comment (altmannyarden) [inline docs/indicators/domain-extraction.md:6]: regex = regular expression

File: docs/indicators/domain-extraction.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+In this documentation, we will elaborate on the Domain indicator components and what output you should expect when extracting indicators of type Domain.
```
Comment (altmannyarden) [inline docs/indicators/domain-extraction.md:7]: Domain indicator extraction components

File: docs/indicators/domain-extraction.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- An Escaped and Unescaped Domains
+
+The regex can extract Domains from one of the following:
+- Explicit Domain
+- URL
+- Email Address
+
+### Formatting Script
+
+After extracting the Domain using regex, an ‘ExtractDomainAndFQDNFromUrlAndEmail’ formatting script iterates on each given Domain and does the following:
```
Comment (altmannyarden) [inline docs/indicators/domain-extraction.md:28]: regex to regular expression

File: docs/indicators/domain-extraction.md (hunk 1 above)
Comment (melamedbn) [reply docs/indicators/domain-extraction.md:6]: domain-extraction.md isn't related, please ignore.

File: docs/indicators/domain-extraction.md (hunk 3 above)
Comment (melamedbn) [reply docs/indicators/domain-extraction.md:28]: domain-extraction.md isn't related, please ignore.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**